.github/workflows/pr_title.yml

```diff
@@ -0,0 +1,72 @@
+name: "Lint PR Title"
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+      - reopened
+
+permissions:
+  pull-requests: write
+
+jobs:
+  main:
+    name: Validate PR title
+    runs-on: ubuntu-latest
+    steps:
+      - uses: amannn/action-semantic-pull-request@v5
+        id: lint_pr_title
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          types: |
+            build
+            ci
+            docs
+            feat
+            fix
+            chore
+            refactor
+            style
+            test
+            perf
+          scopes: |
```

Comment:
For scopes, let's just have the following:

projects, typo, workflow, blueprint, security